src/main/webapp/webpack.config.js

```diff
@@ -158,5 +163,12 @@ module.exports = (env, argv) => {
         writeToDisk: true,
       }),
     ],
-  };
+  });
+
+  config.plugins.push(new MiniCssExtractPlugin({
+    ignoreOrder: true,
+    filename: "css/[name]-[contenthash].css",
```

Comment:
I also decided to change the naming convention of our generated css to match naming of js.

Reply:
That makes some sense.

I still hate how large our output files are.  Why is Ant Design so heavy (not your fault).